pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -572,5 +572,6 @@ in with passthru; stdenv.mkDerivation {
     license = licenses.psfl;
     platforms = platforms.linux ++ platforms.darwin;
     maintainers = with maintainers; [ fridh ];
+    mainProgram = "python3";
```

Comment:
This is not quite correct, fix in #247955

Reply:
It preserved the prior behavior of `lib.getExe` on python, by virtue of being equal to the `pname` value (except for `python3Minimal`, where `"python3"` is a fix).

I have no opinion on what the `mainProgram` for the python interpreters should be.